crypto/rand/rand_unix.c

```diff
@@ -81,7 +81,8 @@ static uint64_t get_timer_bits(void);
 #   define OSSL_POSIX_TIMER_OKAY
 #  endif
 # endif
-#endif /* defined(OPENSSL_SYS_UNIX) || defined(__DJGPP__) */
+#endif /* (defined(OPENSSL_SYS_UNIX) && !defined(OPENSSL_SYS_VXWORKS))
+          || defined(__DJGPP__) */
```

Comment:
_FWIW @levitte: when checking #ifdef indentation with Emacs, its lax interpretation of whitespace (\<space> matches any sequence of whitespace characters) can become quite annoying. To turn it off, you can add the following lines to your .emacs file:_

```
(setq isearch-lax-whitespace nil)
(setq isearch-regexp-lax-whitespace nil)
```

Reply:
I'll complete the pull request later, as soon as I figured out the correct location for the typedefs from e_os.h in the #ifdef spaghetti of rand_unix.c